python/tests/integration/arcticdb/test_arctic.py

```diff
@@ -1335,3 +1337,28 @@ def test_norm_failure_error_message(arctic_library):
         write_batch_exception.value
     )
     assert all("write_pickle" not in str(e.value) for e in [append_exception, append_batch_exception, update_exception])
+
+
+def create_library(uri, lib_name):
+    ac = Arctic(uri)
+    ac.create_library(lib_name)
+    assert lib_name in ac.list_libraries()
+
+
+@pytest.mark.parametrize(
+    "multiprocess", [
+        "spawn",
+        pytest.param("fork", marks=FORK_SUPPORTED),
+        pytest.param("forkserver", marks=FORK_SUPPORTED),
+    ]
+)
+def test_s3_checksum_off_by_env_var(s3_storage, lib_name, multiprocess):
```

Comment:
Should check that the fixture actually fails in the way you expect when the checksum is enabled